### PR TITLE
[RFR] Remove heavy dependencies from dicom and fileformats

### PR DESCRIPTION
**Pull request by SimonBiggs**
@kingrpaul to be able to achieve the following:

https://github.com/pymedphys/pymedphys/issues/223

I needed to remove the Pillow dependency. Pillow is quite a heavy dependency, so I moved narrow_png to labs. See the following issue:

https://github.com/pymedphys/pymedphys/issues/253

Hope this is okay...

`pymedphys_dicom` depends on `pymedphys_analysis`, which is why psutil and Pillow needed stripping out.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -8,16 +8,17 @@
     "docker:internal": "yarn install:prod",
     "git:clean": "git clean -fxd",
     "git:undo": "git reset --soft HEAD~1",
-    "install:dev": "yarn lerna run pip:install:dev && pip install -e .",
-    "install:prod": "yarn lerna run pip:install:prod && pip install .",
+    "install:dev": "yarn lerna run pip:install:dev --concurrency 1 && pip install -e .",
+    "install:prod": "yarn lerna run pip:install:prod --concurrency 1 && pip install .",
```
Comment (SimonBiggs) [inline package.json]: I believe this fixes issues with running `yarn bootrap:dev:all`

**Comment by SimonBiggs** [comment]
Let's merge, I'll need to depend things on this in the near future...

**Comment by SimonBiggs** [comment]
Thanks @kingrpaul :)

**Comment by kingrpaul** [comment]
> 
> 
> Thanks @kingrpaul :)

You're welcome @SimonBiggs. Though having some difficulties, was pretty much in the process of moving most (all?) of mine into "labs". Difficulties may pass when you finish the changes you are working on. Expaning but also putting a GUI in front of things so that I can use / share more easily, but trying to make sure will integrate with pymedphys-GUI when that is settled.

**Comment by SimonBiggs** [comment]
You've confused me a little there. Can you explain that again

**Comment by kingrpaul** [comment]
Sorry, I used too much short-hand. 

I am already moving most or all of my stuff into the labs. So, it is not a problem for me that you find it necessary to move something into the labs. In addition to expanding the functions, I am setting up a user interface, to make these things more useful to me. But, I hope to find that they will integrate easily with the pymedphys user interface, when that is developed. I am having some challenges in moving things to the lab. But, I know that you are making major structural changes, so I hope that my difficulties will lessen after those changes are complete. 

I think I missed part of the discussion, though. So, I am not sure what is being sought in making the structural changed that you are making. 

Hoping that was less cryptic.

**Comment by SimonBiggs** [comment]
Yup, that makes heaps more sense :). The main reasoning for splitting the
back end up into subpackages is that in the rare case where environments
cannot pull in two many dependencies, it is possible to still use portions
of pymedphys without having to install all of the dependencies.

This is particularly important for pyodide, where the options of which
dependencies can be used are currently quite small.

The greater majority of users of pymedphys will be none the wiser as the
main package, pymedphys, still exports the same interface.

On that note, if you're finding it difficult to move packages to the labs I
could do that for you if you would like?

**Pull request merged**